### PR TITLE
Bump phpstan to level 1

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: src/Utility/Xml.php [hunk 1]
```diff
@@ -328,6 +328,7 @@ protected static function _fromArray($dom, $node, &$data, $format)
      */
     protected static function _createChild($data)
     {
+        $value = $dom = $key = $format = $node = null;
```
Comment (dereuromark) [inline src/Utility/Xml.php]: I would just get rid of the remaining extract()s in one of the next PRs/commits.

**Comment by markstory** [comment]
PHPStan builds are failing.

**Comment by antograssiot** [comment]
It is because #10375 has been merged meanwhile

**Comment by ADmad** [comment]
Thanks @antograssiot for the additional fixes.

**Pull request merged**